ee/vellum_ee/workflows/display/workflows/vellum_workflow_display.py

```diff
@@ -170,15 +171,19 @@ def serialize(self) -> JsonObject:
                 )
 
                 if source_node_display:
+                    if isinstance(source_node_display, BaseNodeVellumDisplay):
+                        source_handle_id = source_node_display.get_source_handle_id(
+                            port_displays=self.display_context.port_displays
+                        )
+                    else:
+                        default_port = next(k for k in source_node_display.port_displays.keys() if k.default)
+                        source_handle_id = source_node_display.port_displays[default_port].id
+
                     synthetic_output_edges.append(
                         {
                             "id": str(workflow_output_display.edge_id),
                             "source_node_id": str(source_node_display.node_id),
-                            "source_handle_id": str(
-                                source_node_display.get_source_handle_id(
-                                    port_displays=self.display_context.port_displays
-                                )
-                            ),
+                            "source_handle_id": str(source_handle_id),
```

Comment:
note that this means even before this PR, we don't support synthetic output edges from non-default ports